lizmap/modules/admin/locales/fr_FR/admin.UTF-8.properties

```diff
@@ -26,6 +26,7 @@ configuration.services.proxyMethod.label=Envoi des requêtes à QGIS Server avec
 configuration.services.debugMode.label=Mode de débogage
 configuration.services.allowUserAccountRequests.label=Autoriser les visiteurs à demander un compte
 configuration.services.allowUserAccountRequests.help.deactivated=Cette option est désactivée parce que l'authentification ldap est utilisée
+config.services.allowUserAccountRequest.noemail = Vous ne pouvez pas modifier cette option sans avoir défini d'adresse d'expédition d'email
```

Comment:
It should be : "Cette option ne peut pas être activée car l'adresse email d'expedition n'est pas définie dans la configuration de Lizmap. Contactez l'administrateur de l'application."

Same in english